src/Request/Request.cpp

```diff
@@ -199,5 +239,5 @@ Request::~Request ( )
 //----------------------------------------------------- Méthodes protégées
 void Request::swap(Request& other)
 {
-  
+  *this = other;
```

Comment:
use std::swap on each member ie. 
```suggestion
  using std::swap;
  
  swap(first.ipAddress, second.ipAddress);
  //etc..
```